moz.build needs sorted lists

This must be a recent change, but I'm getting build errors now, [such as this build](https://treeherder.mozilla.org/#/jobs?repo=try&revision=2adf735fe1686490e288b33e699ea5b92d13a9e5&selectedJob=146538732) with an error: 

```
[task 2017-11-21T20:23:13.501Z] 20:23:13     INFO -  The error occurred while processing the following file:
[task 2017-11-21T20:23:13.502Z] 20:23:13     INFO -      /builds/worker/workspace/build/src/browser/extensions/screenshots/moz.build
[task 2017-11-21T20:23:13.502Z] 20:23:13     INFO -  The error was triggered on line 29 of this file:
[task 2017-11-21T20:23:13.502Z] 20:23:13     INFO -      'webextension/sitehelper.js'
[task 2017-11-21T20:23:13.502Z] 20:23:13     INFO -  An error was encountered as part of executing the file itself. The error appears to be the fault of the script.
[task 2017-11-21T20:23:13.502Z] 20:23:13     INFO -  The error as reported by Python is:
[task 2017-11-21T20:23:13.502Z] 20:23:13     INFO -      ['UnsortedError: An attempt was made to add an unsorted sequence to a list. The incoming list is unsorted starting at element 0. We expected "webextension/assertIsBlankDocument.js" but got "webextension/domainFromUrl.js"\n']
[task 2017-11-21T20:23:13.901Z] 20:23:13     INFO -  *** Fix above errors and then restart with\
[task 2017-11-21T20:23:13.901Z] 20:23:13     INFO -                 "/usr/local/bin/gmake -f client.mk build"
[task 2017-11-21T20:23:13.901Z] 20:23:13     INFO -  client.mk:145: recipe for target 'configure' failed
```

Sorting the lists (manually) in moz.build seems to fix the error. We should update `mc_export.py`. This is reproducible locally with `./mach build`

--- a/bin/update_mozbuild.py
+++ b/bin/update_mozbuild.py
@@ -42,7 +42,7 @@ def rewriteMozBuild(outputLoc, fileList):
             insertion_text += \
                 "FINAL_TARGET_FILES.features['screenshots@mozilla.org']%s += [\n" % mozBuildPathName + \
                 "  '" + \
-                "',\n  '".join(fileList[dir]) + "'\n]\n\n"
+                "',\n  '".join(sorted(fileList[dir])) + "'\n]\n\n"
 
         new_contents = re.sub(
             '# AUTOMATIC INSERTION START(.*)# AUTOMATIC INSERTION END',

